### DISCUSSION TITLE
[N64] Is there a tool to check which save type a N64 ROM uses?

### DISCUSSION THREAD

**Discussion by KobeOne2** [sanni/cartreader#971] [Need help]
I'm not able to find anything using Google so I'm asking here. The save type (EEPROM, SRAM, Flash) for most official games can be easily found using the link provided in the wiki but for ROM hacks this is a bit more difficult.

For example there is a console compatible SM64 Star Road hack but I'm not able to find anything on the save type used. Also the save generated by e.g. RetroArch seems to always be 297.0 kB no matter the save type.

**Comment by sanni** [comment]
Your best bet is a low level emulator like cen64, ares, mame etc. that should create the proper save files.
